src/components/Training/Formats.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import styled from 'styled-components';
+import breakpoint from 'styled-components-breakpoint';
+
+import { SectionTitle, Subtitle, BodyPrimary } from '../Typography';
+import { Grid, Row, Col } from '../grid';
+import BlueBackground from '../Common/BlueBackground';
+import CustomisedBulletpoint from '../Common/CustomisedBulletpoint';
+
+const TitleRow = styled(Row)`
+  padding-top: ${({ theme }) => theme.space[4]};
+  padding-bottom: ${({ theme }) => theme.space[4]};
+
+  ${breakpoint('tablet')`
+    padding-top: ${({ theme }) => theme.space[6]};
+    padding-bottom: ${({ theme }) => theme.space[6]}
+  `}
+`;
+
+const FormatsRow = styled(Row)`
+  padding-bottom: ${({ theme }) => theme.space[5]};
+
+  ${breakpoint('tablet')`
+    padding-bottom: ${({ theme }) => theme.space[7]}
+  `};
+`;
+
+const FormatCol = styled(Col)`
+  padding-bottom: ${({ theme }) => theme.space[4]};
+
+  ${breakpoint('desktop')`
+    padding-bottom: ${({ theme }) => theme.space[0]}
+  `};
+`;
+
+const Icon = styled.img`
+  padding-bottom: 2;
+`;
+
+const Bullets = styled.div`
+  padding-top: 2;
+  max-width: 80%;
+`;
+
+const Formats = ({ formats }) => (
+  <BlueBackground>
+    <Grid>
+      <TitleRow>
+        <Col width={[1]}>
+          <SectionTitle reverse>Training formats</SectionTitle>
+          <BodyPrimary muted reverse>
+            We thoughtfully tailor training based on your needs and expertise.
+          </BodyPrimary>
+        </Col>
+      </TitleRow>
+      <FormatsRow>
+        {formats.map(format => (
+          <FormatCol width={[1, 1, 1, 1, 1 / 2, 1 / 2, 1 / 3]} key={format.id}>
+            <Icon
```

Comment:
Create an ImageWrapper component like you've used here: 

https://github.com/yldio/yld.io/pull/427/files#diff-02ae2f34568c66a2ade52d49b0ffaf7dR28

Don't set margin or paddings on the img itself as this can change the image dimensions.

Reply:
In this instance you've lost the padding bottom that exists on production. 

Production: 
![image](https://user-images.githubusercontent.com/7108120/68866722-bd86d480-06ec-11ea-847d-bd8f061bec87.png)

Your branch: 
![image](https://user-images.githubusercontent.com/7108120/68866753-c7103c80-06ec-11ea-8be7-b67f413264f8.png)